frontend/src/App.vue

```diff
@@ -382,9 +411,87 @@ export default defineComponent({
         list.pop()
       }
     },
-    getFlagEmoji
+    getFlagEmoji,
+    updateConnectionStatus() {
+      this.isOffline = !window.navigator.onLine
+    },
+    async subscribeToPush() {
```

Comment:
Nach meinem Empfinden würde es Sinn ergeben, alle Funktionen, die nicht auf Attribute von App angewiesen sind, in eine extra Datei auszulagern und dann zu imporitieren